source/material_model/rheology/frank_kamenetskii.cc

```diff
@@ -81,14 +79,29 @@ namespace aspect
                            "for a total of N+1 values, where N is the number of all compositional fields or only "
                            "those corresponding to chemical compositions. "
                            "If only one value is given, then all use the same value.  Units: None");
-
         prm.declare_entry ("Pressure prefactors for Frank Kamenetskii", "0.0",
                            Patterns::List(Patterns::Double (0.)),
                            "A prefactor for the pressure term in the viscosity approximation, "
                            "for a total of N+1 values, where N is the number of all compositional fields or only "
                            "those corresponding to chemical compositions. "
                            "If only one value is given, then all use the same value. "
                            "Units: None");
+        prm.declare_entry ("Reference Temperature for Frank Kamenetskii", "0.0",
+                           Patterns::List(Patterns::Double (0.)),
+                           "A reference temperature in the viscosity approximation which "
+                           "specifies where the FK temperature dependence goes to 0. "
+                           "Given for a total of N+1 values, where N is the number of all compositional fields "
+                           "or only those corresponding to chemical compositions. If only one "
+                           "value is given, then all use the same value. "
+                           "Units: K");
+        prm.declare_entry ("Reference Pressure for Frank Kamenetskii", "0.0",
+                           Patterns::List(Patterns::Double (0.)),
```

Comment:
same comment as above, `Reference pressures for Frank Kamenetskii` and `std::numeric_limits<double>::max()`